### DISCUSSION TITLE
Gibbs free energy calculation

### DISCUSSION THREAD

**Discussion by TimoSommer** [ACEsuit/mace#718] [Q&A] [answered]
Hello,
thank you very much for this great package! I am interested in using MACE (after fitting it) to relax the geometry of list of single molecules and then calculating their free energy corrections in addition to their eletronic energy. Essentially, I want to use MACE the same way I would Gaussian, just faster: a DFT geometry relaxation with subsequent calculation of the free energy of a single molecule. 

I have searched through the documentation, the source code and some papers, but I have not really found an example that covers this. All I have found is that using openmm it seems to be possible to calculate the Gibbs free energy with MACE, as for example in the following paper: https://arxiv.org/abs/2405.18171
However, I have no experience with openmm, I only know its made more for molecular simulations with force-fields than for ab-initio calculations. Therefore, I find it hard to tell how difficult my problem is to implement and how I would approach this. Should I use openmm, but get rid of all molecular simulations features such as temperature? Or is the Gibbs free energy maybe implemented in the MACE ASE calculator?

**Comment by gabor1** [comment]
You can use ASE to do this task, but you need to write the script yourself. ASE can take your input geometry, relax it (I recommend the PreconLBFGS optimiser), then you can use the phonon calculation packages that ASE supports to compute the vibrational frequencies, I think some of them return to you the free energy directly (by calculating the vibrational entropy).

**Comment by gabor1** [comment, marked as answer]
there are several options to do with which optimiser to use, what tolerances to optimise the geometry to, how to check if the optimisation went wrong etc. with the vibrational calculation again there are options. the latest MACE can calculate the Hessian analytically for example, and then just a diagonalisation will return the vibrational eigenvalues.

**Reply by TimoSommer** [reply]
Thank you very much, that sounds good! I will let you know how it goes if and when I get around to try this out.
